cg/store/models.py

```diff
@@ -519,6 +519,9 @@ class Flowcell(Model):
     updated_at = Column(types.DateTime, onupdate=dt.datetime.now)
 
     samples = orm.relationship("Sample", secondary=flowcell_sample, backref="flowcells")
+    sample_lane_sequencing_metrics = orm.relationship(
+        "SampleLaneSequencingMetrics", backref="flowcell", lazy="dynamic"
```

Comment:
```suggestion
        "SampleLaneSequencingMetrics", back_populates="flowcell", lazy="dynamic"
```

backref is a legacy parameter:
https://docs.sqlalchemy.org/en/20/orm/backref.html